gcc/config/riscv/riscv-ftypes.def

```diff
@@ -30,8 +30,24 @@ DEF_RISCV_FTYPE (0, (USI))
 DEF_RISCV_FTYPE (0, (VOID))
 DEF_RISCV_FTYPE (1, (VOID, USI))
 DEF_RISCV_FTYPE (1, (VOID, VOID_PTR))
+//DEF_RISCV_FTYPE (1, (USI, VOID_PTR))            //RISCV_USI_FTYPE_VOID_PTR
```

Comment:
Need to clean this up.

Here I was trying to figure out which types we need for ALU *only*.